docs/docs-beta/docs/guides/metadata.md

```diff
@@ -116,9 +116,18 @@ Dagster+ provides rich visualization and navigation of column lineage in the Ass
 
 ## Linking assets with source code \{#source-code}
 
-import Experimental from '../partials/\_Experimental.md';
+import Preview from '../partials/\_Preview.md';
+
+<Preview />
+
+import Beta from '../partials/\_Beta.md';
+
+<Beta />
+
+import Superseded from '../partials/\_Superseded.md';
+
+<Superseded />
```

Comment:
Should we also include `Deprecated`. My understanding is that `General Adoption` should be the only state without a warning.

Reply:
Yeah, I think we should include `Deprecated`, although the distinction between `Superseded` and `Deprecated` is a bit unclear to me. The way `Superseded` is defined here sounds like how `Deprecated` is commonly used in a lot of other software.